docs/developers/connectors/index.md

```diff
@@ -4,38 +4,24 @@ Exchange connectors are packages of code that link Hummingbot's internal trading
 
 ## Examples / templates
 
-There are [existing connectors](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector). Note that each folder contained here marks different exchange connector types. These should serve as a template for creating a new exchange connector.
+You can find the existing connectors [here](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector).
+Note that each folder contained here marks different exchange connector types. These should serve as a template for creating a new exchange connector.
 
 Building a new exchange connector requires conforming to the template code to the new exchange's APIs, identifying and handling any differences in functions/behaviors, and testing the new exchange connector on that exchange.
 
-The following list some examples/templates that you can refer to when building the connector:
+## The Gold Standard
 
-### Spot Connectors
-
-- [Ndax](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/ndax)
-- [Crypto.com](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/crypto_com) 
-
-### Perpetual Connectors
-
-- [Bybit Perpetual](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/derivative/bybit_perpetual)
+We recommend referring to the [Binance Spot](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/binance)
+and the [Binance Spot](https://github.com/hummingbot/hummingbot/tree/master/hummingbot/connector/exchange/binance)
+connectors as the gold standard and most up-to-date implementations in the code base.
```

Comment:
- Add something similar telling devs building perp connectors to use Binance Perpetual